diffyscan/diffyscan.py

```diff
@@ -105,15 +105,15 @@ def run_source_diff(
 ):
     logger.divider()
     logger.okay("Contract", contract_address_from_config)
-    logger.okay("Blockchain explorer Hostname", config["explorer_hostname"])
+    logger.okay("Blockchain explorer Hostname", get_explorer_hostname(config))
```

Comment:
Store the result in a variable so the function doesn't need to be run a second time, or see comment below